aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.4.0
+version: 5.4.1
```

```suggestion
version: 5.5.0
```
According to semver, this is new feature.

---

yes, sorry. Done